README.md

```diff
@@ -1,3 +1,80 @@
 # Fediverso Seguro
 
-Uma lista de instâncias do Fediverso que compartilham o mesmo conjunto de regras.
\ No newline at end of file
+Uma lista de instâncias do Fediverso que compartilham o mesmo conjunto de regras.
+
+
+## Regras da Instância
+
+Código de conduta
+-----------------
+
+### Comportamenos inadmissíveis
+
+**Contas que engajarem nos seguintes padrões de comportamento serão suspensas sem recurso:**
+
+*   Publicação de conteúdo ilegal, ou ainda sem definição legal concreta, como: Pornografia infantil.
+*   Conteúdo que fetichiza menores de idade (i.e. lolicon, shotacon, novinhas).
+*   Apologia ao nazismo ou fascismo.
+*   Discurso que incentiva práticas nocivas ao bem-estar físico ou mental como distúrbios alimentares (pró-ana),pró-mia), automutilação (pro-cutting) e suicídio.
+
+**Discurso de ódio e/ou inflamatório, como:**
+
+*   Comportamentos e discursos preconceituosos como: racismo, sexismo, preconceito direcionado à sexualidade, preconceito contra identidade ou expressão de gênero, capacitismo, xenofobia, islamofobia, antissemitismo, gordofobia.
+*   Ameaças de violência ou linguagem violenta direcionada a outras pessoas.
+*   Apologia a governos totalitários.
+*   Xenofobia e/ou nacionalismo extremista.
+*   Desinformação intencional ("fake news").
+*   Intolerância religiosa.
+
+**Ameaça de publicação ou publicação de conteúdo privado, como:**
+
+*   Informações pessoais identificáveis (prática conhecida como "doxing").
+*   Conversas privadas, como mensagens privadas ou diretas, sem o consentimento da outra parte.
+
+**Atitudes invasivas e/ou intimidadoras, como:**
+
+*   Fazer comentários ou perguntas invasivas (tais como "qual é o seu nome de registro?" ou insistir em saber a localização de determinada pessoa).
+*   Insistir em continuar uma conversa após a outra parte expressamente pedir para que ela se encerre (assédio).
+*   Atenção sexual indesejada. Isso inclui piadas ou comentários sexuais; e avanços sexuais indesejados.
+*   Intimidação deliberada, stalking.
+*   Atenção religiosa indesejada. Isso inclui convidar, insistir ou pregar sobre religiões sem consentimento.
+
+**Falsidade ideológica, como:**
+
+*   Criar contas para se passar por celebridades, marcas ou entidades.
+
+**Política de aviso de conteúdo**
```

Comment:
Sobre política de aviso de conteúdo eu concordo com todos os itens, mas retiraria da parte "conteúdo recomendado" o item "Política". Pois algumas instâncias podem ser focadas em discussões politicas.

E moveria os itens:

- Distúrbios mentais
- Distúrbios alimentares
- Relacionamentos abusivos

para obrigatórios.

Mas também não sei até que pondo o "uso recomendado" é decisão da própria instância ou não

Reply:
"Uso recomendado" me passa a impressão de "olha, preferimos que você use aviso de conteúdo mas se não usar não tem problema", enquanto "obrigatório" pode causar alguma sanção se não for obedecido. Neste caso, acho que política pode permanecer no uso recomendado e mover os outros itens para obrigatórios.